Documentation/contributor-guide/triage_issues.md

```diff
@@ -48,9 +48,42 @@ Adding a `type` label to an issue helps create visibility on the health of the p
+
+ 2. Bug reports or questions about unspported versions of etcd, for example <https://github.com/etcd-io/etcd/issues/15796>. When responding to these issues please refer to our [supported versions documentation](https://etcd.io/docs/latest/op-guide/versioning) and encourage the reporter to upgrade to a recent patch release of a supported version as soon as possible. We should limit the effort supporting users that do not make the effort to run a supported version of etcd or ensure their version is patched.
+
+ 3. Bug reports that do not provide a complete list of steps to reproduce issue and/or contributors are not able to reproduce the issue, for example <https://github.com/etcd-io/etcd/issues/15740>. We should limit the effort we put into reproducing issues ourselves and motivate users to provide necessary information to accept the bug report.
+
+ 4. General questions that are filed using feature request or bug report issue templates, for example <https://github.com/etcd-io/etcd/issues/15914>. Note - These types of requests may surface good additions to our [frequently asked questions](https://etcd.io/docs/v3.5/faq).
+
+If you identify that an issue is a support request please:
+
+ 1. Add the `type/support` or `type/question` label.
+
+ 2. Add the following comment to inform the issue creator that discussions should be used instead and that this issue will be converted to a discussion.
+
+    > Thank you for your question, this support issue will be moved to our [Discussion Forums](https://github.com/etcd-io/etcd/discussions).
+    >
+    > We are trying to consolidate the channels to which questions for help/support are posted so that we can improve our efficiency in responding to your requests, and to make it easier for you to find answers to frequently asked questions and how to address common use cases.
+    >
+    > We regularly see messages posted in multiple forums, with the full response thread only in one place or, worse, spread across multiple forums. Also, the large volume of support issues on GitHub is making it difficult for us to use issues to identify real bugs.
+    >
+    > Members of the etcd community use Discussion Forums to field support requests. Before posting a new question, please search these for answers to similar questions, and also familiarize yourself with:
+    >
+    > 1. [user documentation](https://etcd.io/docs/latest)
+    > 2. [frequently asked questions](https://etcd.io/docs/v3.5/faq)
+    >  
+    > Again, thanks for using etcd and raising this question.
+    >
+    > The etcd team
+
+ 3. Finally, click `Convert to discussion` on the right hand panel, selecting the appropriate discussion category.
```

Comment:
qq, does step 2 and 3 has a grace period before converting to discussion? Or what's the downside of using github discussion for `type/support` and `type/issues` issues?

Reply:
That's a good question Chao - I was not proposing to use a grace period because we do have the ability to convert back if we make a mistake or we determine that a valid bug or feature has been identified, refer https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue#creating-an-issue-from-discussion 

I have no objection to introducing a grace period before conversion if we find that these initial guidelines are too trigger happy with conversion. Let's see how it goes 🙏🏻

Reply:
> https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue#creating-an-issue-from-discussion

Nice!! I think that's good enough.